Dockerfile

```diff
@@ -30,9 +30,9 @@ RUN npm install -g grunt-cli
 RUN npm install --global grunt
 
 RUN echo '{ "allow_root": true }' > /root/.bowerrc
-RUN cd mbofui && bower install && npm install && grunt --gruntfile Gruntfile-docker.js
+RUN cd mbofui && bower install && npm install && grunt docker
```

Comment:
I like having to only type grunt docker instead of using the file option. Makes it more readable!